apps/web-giddh/src/app/store/Inventory/inventory.reducer.ts

```diff
@@ -459,14 +459,8 @@ export function InventoryReducer(state: InventoryState = initialState, action: C
             let updateStockResp: BaseResponse<StockDetailResponse, CreateStockRequest> = action.payload;
             if (updateStockResp.status === 'success') {
                 groupArray = _.cloneDeep(state.groupsWithStocks);
-                let activeGroup = _.cloneDeep(state.activeGroup) || updateStockResp.body.stockGroup;
-                let stateActiveGrp: StockGroupResponse = null;
-                let myGrp = removeStockItemAndReturnIt(groupArray, activeGroup.uniqueName, updateStockResp.queryString.stockUniqueName, null);
-                if (myGrp) {
-                    myGrp.name = updateStockResp.body.name;
-                    myGrp.uniqueName = updateStockResp.body.uniqueName;
-                    addStockItemAtIndex(groupArray, activeGroup.uniqueName, myGrp);
-                }
+                let activeGroupUniqueName = _.cloneDeep(updateStockResp.queryString.stockGroupUniqueName);
+                updateStockIteminGroupArray(groupArray,activeGroupUniqueName, updateStockResp);
```

Comment:
Incorrect camelCase name
```suggestion
                updateStockItemInGroupArray(groupArray,activeGroupUniqueName, updateStockResp);
```